pkg/windows/create_build_env.ps1

```diff
@@ -113,6 +114,11 @@ if (!($port_git)) {
 set-location C:\Temp
 Import-Module "$env:ChocolateyInstall\helpers\chocolateyInstaller.psm1" -Force;
 $ChocoTools = Get-ToolsLocation
+
+if ([string]::IsNullOrEmpty($ChocoTools)) {
+    $ChocoTools = [System.Environment]::GetEnvironmentVariable("ChocolateyToolsLocation","User")
```

Comment:
In my case, $env:ChocolateyInstall seems to be un-defined. So I am getting error in importing module in line 114/115. Consequently, $ChocoTools is resolving to an empty string. 
 However, "ChocolateyToolsLocation" is defined under user variables. So we can use it as a fallback option.